integrations/symfony-bundle.rst

```diff
@@ -342,6 +343,30 @@ The only steps they need is ``require`` one of the adapter implementations in
 their projects ``composer.json`` and instantiating the ``HttplugBundle`` in
 their kernel.
 
+Mock responses in functional tests
+``````````````````````````````````
+To mock responses in your functional tests, proceed as follow:
```

Comment:
s/proceed as follow/use the mock client factory in your test environement configuration/